python/lsst/obs/monocam/monocam.py

```diff
@@ -0,0 +1,219 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the LSST License Statement and 
+# the GNU General Public License along with this program.  If not, 
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import numpy
+import lsst.afw.cameraGeom as cameraGeom
+import lsst.afw.geom as afwGeom
+from lsst.afw.table import AmpInfoCatalog, AmpInfoTable, LL
+from lsst.afw.cameraGeom.cameraFactory import makeDetector
+
+class Monocam(cameraGeom.Camera):
+    """The monocam Camera
+
+    There is one ccd with name "0"
+    It has sixteen amplifiers with names like "00".."07" and "10".."17"
+
+    Standard keys are:
+    amp: amplifier name: one of 00, 01, 02, 03, 04, 05, 06, 07, 10, 11, 12, 13, 14, 15, 16, 17
+    ccd: ccd name: always 0
+    visit: exposure number; this will be provided by the DAQ
+    """
+    # Taken from fit4_20160413-154303.pdf
+    gain = {
+            (0, 0):3.707220,
+            (1, 0):3.724264,
+            (2, 0):3.758828,
+            (3, 0):3.794040,
+            (4, 0):3.724264,
+            (5, 0):3.776352,
+            (6, 0):3.758828,
+            (7, 0):3.776352,
+            (0, 1):3.724264,
+            (1, 1):3.657009,
+            (2, 1):3.640573,
+            (3, 1):3.624284,
+            (4, 1):3.640573,
+            (5, 1):3.707220,
+            (6, 1):3.673594,
+            (7, 1):3.640573,}
+
+    readNoise = {
+                 (0, 0):7.105902,
+                 (1, 0):6.860052,
+                 (2, 0):7.387405,
+                 (3, 0):7.222204,
+                 (4, 0):7.250763,
+                 (5, 0):7.315250,
+                 (6, 0):7.104838,
+                 (7, 0):7.272336,
+                 (0, 1):8.485139,
+                 (1, 1):8.022778,
+                 (2, 1):8.157399,
+                 (3, 1):8.021112,
+                 (4, 1):8.015486,
+                 (5, 1):7.928829,
+                 (6, 1):16.031720,
+                 (7, 1):7.938155,}
+
+
+    def __init__(self):
+        """Construct a TestCamera
+        """
+        plateScale = afwGeom.Angle(13.55, afwGeom.arcseconds) # plate scale, in angle on sky/mm
+        radialDistortion = 0. # radial distortion in mm/rad^2
+        radialCoeff = numpy.array((0.0, 1.0, 0.0, radialDistortion)) / plateScale.asRadians()
+        focalPlaneToPupil = afwGeom.RadialXYTransform(radialCoeff)
+        pupilToFocalPlane = afwGeom.InvertedXYTransform(focalPlaneToPupil)
+        cameraTransformMap = cameraGeom.CameraTransformMap(cameraGeom.FOCAL_PLANE,
+            {cameraGeom.PUPIL: pupilToFocalPlane})
+        detectorList = self._makeDetectorList(pupilToFocalPlane, plateScale)
+        cameraGeom.Camera.__init__(self, "monocam", detectorList, cameraTransformMap)
+
+    def _makeDetectorList(self, focalPlaneToPupil, plateScale):
+        """Make a list of detectors
```

Comment:
I couldn't find this in the developer guide.  Is it just this `"""!Make a list of detectors`?